### ISSUE TITLE
Numbering list is not friendly with images inlined with it

### ISSUE THREAD

**Issue by leolorenzoluis** [TryGhost/Ghost#7183]
### Issue Summary

Using Safari latest version and Mac OSX.
### Steps to Reproduce
1. Type in the following markup:

```
1. Sample content
![]
2. Second sample content
![]
3. Third sample content
![]
```

Any other info e.g. Why do you consider this to be a bug? What did you expect to happen instead?

I'm expecting it to be not reset the number to 1. It should recognized that I placed image and at least the numbering should stay. 

The output would look like this:

```
1. Sample content
[Image]
1. Second sample content
[Image]
1. Third sample content
[Image]
```
### Technical details:
- Ghost Version: 0.9
- Node Version: Latest version
- Browser/OS:  Safari/OSX
- Database: Ghost.db

**Comment by dbalders** [comment]
@leolorenzoluis hey, so to fix this, just put 4 spaces for the images to indent it, and your numbering will be perfect. Ex:

```
1. Sample content
    ![]
2. Second sample content
    ![]
3. Third sample content
    ![]
```

Doing it like that it will work the way you expect. 

Hope that helps!

**Comment by leolorenzoluis** [comment]
@dbalders Thank you it works. However, what about for ``` markup? I tried adding 4 spaces and it didn't indent.

**Comment by dbalders** [comment]
ya, doesn't look like that works. I'll re-open for now

**Comment by iMicknl** [comment]
Is this bug still on the radar? Or is it already solved in the 1.0.0 version. It is quite hard to write technical guides on Ghost now, since you can't use 4 spaces to indent after a ``` code block. 👍
